app/src/renderer/js/main.js

```diff
@@ -341,8 +341,8 @@ document.addEventListener('DOMContentLoaded', () => {
   swapBtn.addEventListener('click', () => {
     [inputWidth.value, inputHeight.value] = [inputHeight.value, inputWidth.value];
     dimensions.ratio = dimensions.ratio.reverse();
-    inputWidth.dispatchEvent(new Event('input'));
-    inputHeight.dispatchEvent(new Event('input'));
+    inputWidth.dispatchEvent(new CustomEvent('input', {detail: {ignoreRatioLocked: true}}));
```

Comment:
Do we need to wrap it in `detail: {}`? is that standard practise? Otherwise just do `new CustomEvent('input', {ignoreRatioLocked: true})` to reduce complexity

Reply:
I wish, but it has to be under a `detail` key

https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Creating_and_triggering_events